src/Commands/InstallCommand.php

```diff
@@ -19,6 +21,17 @@ public function handle(): int
     {
         $this->call('vendor:publish', ['--tag' => 'envy-config']);
 
+        if (!File::exists('.env.example')) {
+            if (
+                $this->confirm('Do you want to create a .env.example file? This file will be used to store your configuration data.')
+            ) {
+                $envExample =  Http::get('https://raw.githubusercontent.com/laravel/laravel/9.x/.env.example')->body();
```

Comment:
This redundant HTTP call brings a senseless dependency on 3rd-party service availability. 
Also, the code fragment will try to download `.env.example` of `9.x` version, while the package is compatible with 8.73 too.